testing/tests/DevExpress.ui.widgets/button.tests.js

```diff
@@ -113,6 +116,22 @@ QUnit.module('options changed callbacks', {
         assert.strictEqual(template.firstCall.args[0].custom, 1, 'custom field is correct');
     });
 
+    QUnit.test('stylingMode', function(assert) {
+        assert.ok(this.element.hasClass(BUTTON_CONTAINED_STYLE_CLASS));
+
+        this.instance.option('stylingMode', 'text');
+        assert.ok(this.element.hasClass(BUTTON_TEXT_STYLE_CLASS));
+        assert.notOk(this.element.hasClass(BUTTON_CONTAINED_STYLE_CLASS));
```

Comment:
What about BUTTON_OUTLINED_STYLE_CLASS?

Reply:
I check the widget has the new styling class and does not have the previous styling class. I think it is no reason to check other classes.

Reply:
ok